### DISCUSSION TITLE
Forgot password and henceforth can't log in to my account.

### DISCUSSION THREAD

**Discussion by jeremy-sanchez** [quarto-dev/quarto-cli#3666] [Q&A]
To whom it may concern,

I forgot my password, but I'm having trouble recovering it/logging in to my account. When I attempt to sign in at https://quartopub.com/sign-in using my Google account (my e-mail on the account is through Google), I'm just redirected back to that page. When I try to sign in regularly and note that I don't know my password, once I enter my e-mail again I get a prompt saying that my account is not authorized to recover my password in this manner. I'm not sure what to do; I want to clean up my website which currently has duplicate slide decks so that my students can navigate it easier, but at this point I have no way of getting in, though somehow I can still publish new decks through the terminal. Any assistance would be greatly appreciated!

Blessings,

Jeremy Sanchez

**Comment by dpastoor** [comment]
Hi Jeremy - are you doing Sign in with google

![image](https://user-images.githubusercontent.com/3196313/207719532-d69396b9-1e02-4d89-b4b9-039bc330a59f.png)

in which case you would not use/need a password, or are you saying you used your email but did a regular email/password setup originally?

**Reply by softwarenerd** [reply]
@jeremy-sanchez, could you send me your email address? My email is [brian.lambert@posit.co](mailto:brian.lambert@posit.co).
